src/server/graphql/types/AgendaItem.js

```diff
@@ -0,0 +1,43 @@
+import {GraphQLBoolean, GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql';
+import GraphQLISO8601Type from 'server/graphql/types/GraphQLISO8601Type';
+import TeamMember from 'server/graphql/types/TeamMember';
+
+const AgendaItem = new GraphQLObjectType({
+  name: 'AgendaItem',
+  description: 'A request placeholder that will likely turn into 1 or more tasks',
+  fields: () => ({
+    id: {type: new GraphQLNonNull(GraphQLID), description: 'The unique agenda item id teamId::shortid'},
+    content: {type: new GraphQLNonNull(GraphQLString), description: 'The body of the agenda item'},
+    createdAt: {
+      type: GraphQLISO8601Type,
+      description: 'The timestamp the agenda item was created'
+    },
+    isActive: {
+      type: GraphQLBoolean,
+      description: 'true until the agenda item has been marked isComplete and the meeting has ended'
+    },
+    isComplete: {
+      type: GraphQLBoolean,
+      description: 'true if the agenda item has been addressed in a meeting (will have a strikethrough or similar)'
+    },
+    sortOrder: {
+      type: GraphQLFloat,
+      description: 'The sort order of the agenda item in the list'
+    },
+    teamId: {type: new GraphQLNonNull(GraphQLID), description: '*The team for this agenda item'},
+    teamMemberId: {type: new GraphQLNonNull(GraphQLID), description: 'The teamMemberId that created this agenda item'},
+    updatedAt: {
+      type: GraphQLISO8601Type,
+      description: 'The timestamp the agenda item was updated'
+    },
+    teamMember: {
+      type: TeamMember,
+      description: 'The team member that created the agenda item',
+      resolve: async ({teamMemberId}, args, {dataLoader}) => {
```

Comment:
I _think_ you may be confusing types & fields.
A GraphQL Type is the shape of the object, in this case a team member with an id, picture, name, etc. A type contains no information on how to resolve it.
A _field_ is what you see here, which is a type, description, resolver, and other things we don't use (deprecated reason, default value, etc). 
so 1 agendaItem has 1 teamMember. Each agenda item is guaranteed a teamMemberId foreign key. The resolve is simply saying "Given that foreign key for an agenda item, here's how to get the full TeamMember". Now if you check out User.js, you'll see teamMember exists there as well, but since 1 user has many team members, it takes a teamId arg to determine which team member you want.